main.go

```diff
@@ -307,14 +307,23 @@ func main() {
 	fmt.Println()
 
 	//
-	// app_path_pattern is required, and the apk_path_pattern is deprecated
-	if config.APKPathPattern != "" {
-		log.Warnf("apk_path_pattern input is **DEPRECATED** - use the app_path_pattern input instead.")
-	}
-	if config.APKPathPattern != "" && config.AppPathPattern != "" {
-		log.Warnf("Both apk_path_pattern and app_path_pattern inputs are provided. Using the app_path_pattern only.")
+	// Config migration from apk_path_pattern to app_path_pattern
+	// The apk_path_pattern is deprecated
+	// New input: app_path_pattern
+	// If the apk_path_pattern is used log a warning, and still use the deprecated apk_path_pattern and ignore the new app_path_pattern temporarily
+	var appPatterns []string
+	if strings.TrimSpace(config.APKPathPattern) != "" {
+		log.Warnf(`Step input 'APK location pattern' (apk_path_pattern) is deprectad and will be removed soon,
```

Comment:
pls mention the removal date as well: https://github.com/bitrise-steplib/steps-google-play-deploy/blob/master/config.go#L124

Reply:
Fixed in:
https://github.com/bitrise-steplib/bitrise-step-android-build/pull/11/commits/f3e30022c36ff6300e0cfb996850772b62c7c645